applications/FluidDynamicsApplication/custom_utilities/estimate_dt_utilities.h

```diff
@@ -77,6 +77,7 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) EstimateDtUtility
         mViscousFourier = 0.0;
         mThermalFourier = 0.0;
         mConsiderArtificialDiffusion = false;
+        mConsiderCompressibility = false;
```

Comment:
```suggestion
        mConsiderSoundVelocityInCFL = false;
```
I think that this is more clear and might avoid misunderstandings when solving weakly compressible formulations.

Reply:
Agreed.